gce_workload_cert_refresh/main.go

```diff
@@ -262,6 +266,25 @@ func refreshCreds() error {
 		return fmt.Errorf("Error writing config_status: %v", err)
 	}
 
+	// Handles the edge case where the config values provided at VM creation may be invalid. This ensures
+	// that the symlink directory exists and contains the config_status to surface config errors to the VM.
+	defer func() error {
```

Comment:
a deferred function can't return values; i think you'll need to refactor this a bit to catch the errors you want.

Reply:
Fixed, thanks!